packages/connection-form/src/components/connection-form.tsx

```diff
@@ -622,7 +622,7 @@ function ConnectionForm({
               : formFooterBorderLightModeStyles
           )}
         >
-          <ConnectionFormModalActions
+          <ConnectionFormActions
```

Comment:
☝️ See this is the actions for connection-form, but really for connection-form-modal specifically.